src/viam/components/movement_sensor/client.py

```diff
@@ -134,12 +133,8 @@ async def get_accuracy(
         extra: Optional[Dict[str, Any]] = None,
         timeout: Optional[float] = None,
         **__,
-    ) -> Mapping[str, float]:
-        if extra is None:
-            extra = {}
-        request = GetAccuracyRequest(name=self.name, extra=dict_to_struct(extra))
-        response: GetAccuracyResponse = await self.client.GetAccuracy(request, timeout=timeout)
-        return response.accuracy
+    ) -> MovementSensor.Accuracy:
+        return await self.client.GetAccuracy(GetAccuracyRequest(name=self.name), timeout=timeout)
```

Comment:
Sorry, didn't see on my first pass through but we've lost the `extra` param here as well